src/main/java/core/basesyntax/IllegalArgumentException.java

```diff
@@ -0,0 +1,7 @@
+package core.basesyntax;
+
+class IllegalArgumentException extends RuntimeException {
```

Comment:
delete this file please

Reply:
I deleted on git Hub. It's OK?

Reply:
it's better to know how to do it locally as well, but for now it's enough